tasks/plugins.yml

```diff
@@ -8,11 +8,17 @@
     update: yes
     force: yes
 
-- name: get git hash
+- name: read git commit hash
   shell: git rev-parse --short HEAD
   args:
     chdir: "{{ app_base_path }}/{{ plugin.value.path }}/plugins/{{ plugin.key }}"
   register: p_commit
 
+- name: read git branch name
+  shell: git rev-parse --abbrev-ref HEAD
```

Comment:
From git x.y version (I don't remember exactly) it is also possible to use `git branch --show-current`.
Anyway it's ok like this.